app/portainer/components/datatables/endpoints-datatable/endpointsDatatableController.js

```diff
@@ -66,11 +66,11 @@ angular.module('portainer.app').controller('EndpointsDatatableController', [
     this.setShowIntelInfo = async function () {
       this.settings = await SettingsService.settings();
       const openAMTFeatureFlagValue = this.settings && this.settings.FeatureFlagSettings && this.settings.FeatureFlagSettings['open-amt'];
-      const openAMTFeatureEnabled = this.settings && this.settings.OpenAMTConfiguration && this.settings.OpenAMTConfiguration.Enabled;
+      const openAMTFeatureEnabled = this.settings && this.settings.EnableEdgeComputeFeatures && this.settings.OpenAMTConfiguration && this.settings.OpenAMTConfiguration.Enabled;
```

Comment:
```suggestion
      const openAMTFeatureEnabled = this.settings?.OpenAMTConfiguration?.Enabled;
```

Reply:
I think this is only valid for .ts files 
![image](https://user-images.githubusercontent.com/16310914/148223495-68e9dc86-a352-4978-b72e-166e7ad588f7.png)